ext/nnstreamer/tensor_filter/tensor_filter_nnfw.c

```diff
@@ -504,6 +504,7 @@ nnfw_setInputDim (const GstTensorFilterProperties * prop, void **private_data,
   /** Return -ENOENT till nnfw supports set input dimension internally */
   return -ENOENT;
 
+#if 0
```

Comment:
Do not add ```#if 0```. It deteriorates readability greatly. If these codes are really useless, delete them.

Reply:
I agreed with MJ's opinion.

Reply:
If the below codes are waiting for unimplemented feature, it'd better have some meaningful text along with the ```#if```. e.g., ```#ifndef FEATURE_XXX_IMPLEMENTED```.

Reply:
Got it.